### ISSUE TITLE
KeePassDX old green square theme

### ISSUE THREAD

**Issue by Jon-guy30** [Kunzisoft/KeePassDX#1705]
**Is your feature request related to a problem? Please describe.**
I wasn't sure whether to have this as a feature request or a blank issue but I went with feature request since this is both feedback for the new theme and a request to have the old green square theme back as an option.

**Describe the solution you'd like**
I personally became fond of the old square theme in the previous KeePassDX version. It didn't necessarily fit the theme of my phone but that didn't matter, it worked with the program and was quite functional.

**Describe alternatives you've considered**
I looked around in the app theme options and found a "classic" theme, I thought THAT was gonna be the old theme, but instead it's a strange sort of grey theme I didn't like. I also checked the "system" theme but I was surprised that was paywalled. Why paywall a system theme? I understand having special awesome themes as a bonus for paying patrons, but the system theme? That's just following the OS guidelines, i.e. Material Design 3 (You), nothing special at all. I've would've chosen the system theme to make it follow my phone's colors but couldn't due to the donation paywall. 
If a solid square green theme is not possible due to API changes or whatever I would settle for a solid green theme with the rounded buttons. That's another quirk I don't really like in Material You, the rounded corners thing, again, it would be nice to have the solid green theme back as it was.

**Additional context**
I noticed as I unlocked the database it took a little while longer to unlock the database in the new theme, as if it was somehow slower, or the animation was slower, that might just be my perception but I'd be interested in hearing what others have to say about it. 
I also noticed when I checked my master password all of my typed characters didn't fit in the password field, it's one character short so I have to scroll all the way back to check for typos if I misspelled my password. Previously I could check the whole thing and not have to scroll. My password does fit in the password field but only with the obfuscation, if I click the eye to reveal the password, it doesn't fit anymore.

**Comment by htsmi** [comment]
I agree here, it's a small issue in the scheme of things, but I was very comfortable in the old theme and the new one just seems like too much padding around everything. I'm also not a huge fan of these lists where everything is floating on a rounded card rather than simple dividing lines. I know Google is pushing this kind of design but it's getting to an absurd level, with little islands of text in a sea of whitespace, and almost nothing fits on a single screen. If it's not too much to ask, I know there are probably at least a few others who like the old fashioned simple and compact interfaces.
